### ISSUE TITLE
Version `v1.12.2` in deno.land still uses unstable APIs. Culprit: https://github.com/eta-dev/eta/blob/v1.12.2/deno_dist/file-methods.ts

### ISSUE THREAD

**Issue by avihavai** [eta-dev/eta#126]
Hi! 

The version `v1.12.2` in deno.land still uses unstable APIs ([https://deno.land/x/eta@v1.12.2/file-methods.ts](https://deno.land/x/eta@v1.12.2/file-methods.ts)). The culprit seems to be the tag `v.12.2`, see [https://github.com/eta-dev/eta/blob/v1.12.2/deno_dist/file-methods.ts](https://github.com/eta-dev/eta/blob/v1.12.2/deno_dist/file-methods.ts). Master at [https://github.com/eta-dev/eta/blob/master/deno_dist/file-methods.ts](https://github.com/eta-dev/eta/blob/master/deno_dist/file-methods.ts) is up to date.

This leads to a situation, where programs that use Eta v1.12.2 from deno.land still need to run with the `--unstable` flag.

Output for the command `deno info https://deno.land/x/eta@v1.12.2/mod.ts` (with cleaned cache):

```
Download https://deno.land/x/eta@v1.12.2/mod.ts
Download https://deno.land/x/eta@v1.12.2/parse.ts
Download https://deno.land/x/eta@v1.12.2/compile.ts
Download https://deno.land/x/eta@v1.12.2/compile-string.ts
Download https://deno.land/x/eta@v1.12.2/render.ts
Download https://deno.land/x/eta@v1.12.2/containers.ts
Download https://deno.land/x/eta@v1.12.2/file-handlers.ts
Download https://deno.land/x/eta@v1.12.2/config.ts
Download https://deno.land/x/eta@v1.12.2/file-helpers.ts
Download https://deno.land/x/eta@v1.12.2/polyfills.ts
Download https://deno.land/x/eta@v1.12.2/err.ts
Download https://deno.land/x/eta@v1.12.2/utils.ts
Download https://deno.land/x/eta@v1.12.2/storage.ts
Download https://deno.land/x/eta@v1.12.2/file-utils.ts
Download https://deno.land/x/eta@v1.12.2/file-methods.ts
Download https://deno.land/std@0.66.0/fs/mod.ts
Download https://deno.land/std@0.66.0/path/mod.ts
Download https://deno.land/std@0.66.0/path/glob.ts
Download https://deno.land/std@0.66.0/path/_constants.ts
Download https://deno.land/std@0.66.0/path/_interface.ts
Download https://deno.land/std@0.66.0/path/common.ts
Download https://deno.land/std@0.66.0/path/win32.ts
Download https://deno.land/std@0.66.0/path/posix.ts
Download https://deno.land/std@0.66.0/path/separator.ts
Download https://deno.land/std@0.66.0/fs/ensure_dir.ts
Download https://deno.land/std@0.66.0/fs/empty_dir.ts
Download https://deno.land/std@0.66.0/fs/write_json.ts
Download https://deno.land/std@0.66.0/fs/read_json.ts
Download https://deno.land/std@0.66.0/fs/ensure_file.ts
Download https://deno.land/std@0.66.0/fs/walk.ts
Download https://deno.land/std@0.66.0/fs/eol.ts
Download https://deno.land/std@0.66.0/fs/expand_glob.ts
Download https://deno.land/std@0.66.0/fs/exists.ts
Download https://deno.land/std@0.66.0/fs/move.ts
Download https://deno.land/std@0.66.0/fs/copy.ts
Download https://deno.land/std@0.66.0/fs/ensure_link.ts
Download https://deno.land/std@0.66.0/fs/ensure_symlink.ts
Download https://deno.land/std@0.66.0/fs/_util.ts
Download https://deno.land/std@0.66.0/_util/assert.ts
Download https://deno.land/std@0.66.0/path/_util.ts
local: (my path)
type: TypeScript
dependencies: 39 unique (total 132.91KB)

https://deno.land/x/eta@v1.12.2/mod.ts (684B)
├─┬ https://deno.land/x/eta@v1.12.2/compile-string.ts (3.01KB)
│ ├─┬ https://deno.land/x/eta@v1.12.2/config.ts (4.23KB)
│ │ ├─┬ https://deno.land/x/eta@v1.12.2/compile.ts (1.92KB)
│ │ │ ├── https://deno.land/x/eta@v1.12.2/compile-string.ts *
│ │ │ ├── https://deno.land/x/eta@v1.12.2/config.ts *
│ │ │ ├── https://deno.land/x/eta@v1.12.2/config.ts *
│ │ │ ├── https://deno.land/x/eta@v1.12.2/err.ts (1.29KB)
│ │ │ ├─┬ https://deno.land/x/eta@v1.12.2/file-handlers.ts (9.45KB)
│ │ │ │ ├── https://deno.land/x/eta@v1.12.2/compile.ts *
│ │ │ │ ├── https://deno.land/x/eta@v1.12.2/compile.ts *
│ │ │ │ ├── https://deno.land/x/eta@v1.12.2/config.ts *
│ │ │ │ ├── https://deno.land/x/eta@v1.12.2/config.ts *
│ │ │ │ ├── https://deno.land/x/eta@v1.12.2/err.ts *
│ │ │ │ ├─┬ https://deno.land/x/eta@v1.12.2/file-utils.ts (5.72KB)
│ │ │ │ │ ├── https://deno.land/x/eta@v1.12.2/config.ts *
│ │ │ │ │ ├── https://deno.land/x/eta@v1.12.2/err.ts *
│ │ │ │ │ └─┬ https://deno.land/x/eta@v1.12.2/file-methods.ts (180B)
│ │ │ │ │   ├─┬ https://deno.land/std@0.66.0/fs/mod.ts (480B)
│ │ │ │ │   │ ├─┬ https://deno.land/std@0.66.0/fs/copy.ts (8.45KB)
│ │ │ │ │   │ │ ├── https://deno.land/std@0.66.0/_util/assert.ts (405B)
│ │ │ │ │   │ │ ├─┬ https://deno.land/std@0.66.0/path/mod.ts (717B)
│ │ │ │ │   │ │ │ ├── https://deno.land/std@0.66.0/path/_constants.ts (2.35KB)
│ │ │ │ │   │ │ │ ├── https://deno.land/std@0.66.0/path/_interface.ts (657B)
│ │ │ │ │   │ │ │ ├─┬ https://deno.land/std@0.66.0/path/common.ts (1.14KB)
│ │ │ │ │   │ │ │ │ └─┬ https://deno.land/std@0.66.0/path/separator.ts (264B)
│ │ │ │ │   │ │ │ │   └── https://deno.land/std@0.66.0/path/_constants.ts *
│ │ │ │ │   │ │ │ ├─┬ https://deno.land/std@0.66.0/path/glob.ts (8.12KB)
│ │ │ │ │   │ │ │ │ ├── https://deno.land/std@0.66.0/path/_constants.ts *
│ │ │ │ │   │ │ │ │ ├── https://deno.land/std@0.66.0/path/mod.ts *
│ │ │ │ │   │ │ │ │ └── https://deno.land/std@0.66.0/path/separator.ts *
│ │ │ │ │   │ │ │ ├─┬ https://deno.land/std@0.66.0/path/posix.ts (12.67KB)
│ │ │ │ │   │ │ │ │ ├── https://deno.land/std@0.66.0/path/_constants.ts *
│ │ │ │ │   │ │ │ │ ├── https://deno.land/std@0.66.0/path/_interface.ts *
│ │ │ │ │   │ │ │ │ └─┬ https://deno.land/std@0.66.0/path/_util.ts (3.3KB)
│ │ │ │ │   │ │ │ │   ├── https://deno.land/std@0.66.0/path/_constants.ts *
│ │ │ │ │   │ │ │ │   └── https://deno.land/std@0.66.0/path/_interface.ts *
│ │ │ │ │   │ │ │ ├── https://deno.land/std@0.66.0/path/separator.ts *
│ │ │ │ │   │ │ │ └─┬ https://deno.land/std@0.66.0/path/win32.ts (27.36KB)
│ │ │ │ │   │ │ │   ├── https://deno.land/std@0.66.0/_util/assert.ts *
│ │ │ │ │   │ │ │   ├── https://deno.land/std@0.66.0/path/_constants.ts *
│ │ │ │ │   │ │ │   ├── https://deno.land/std@0.66.0/path/_interface.ts *
│ │ │ │ │   │ │ │   └── https://deno.land/std@0.66.0/path/_util.ts *
│ │ │ │ │   │ │ ├─┬ https://deno.land/std@0.66.0/fs/_util.ts (900B)
│ │ │ │ │   │ │ │ └── https://deno.land/std@0.66.0/path/mod.ts *
│ │ │ │ │   │ │ └─┬ https://deno.land/std@0.66.0/fs/ensure_dir.ts (1.41KB)
│ │ │ │ │   │ │   └── https://deno.land/std@0.66.0/fs/_util.ts *
│ │ │ │ │   │ ├─┬ https://deno.land/std@0.66.0/fs/empty_dir.ts (1.72KB)
│ │ │ │ │   │ │ └── https://deno.land/std@0.66.0/path/mod.ts *
│ │ │ │ │   │ ├── https://deno.land/std@0.66.0/fs/ensure_dir.ts *
│ │ │ │ │   │ ├─┬ https://deno.land/std@0.66.0/fs/ensure_file.ts (1.84KB)
│ │ │ │ │   │ │ ├── https://deno.land/std@0.66.0/path/mod.ts *
│ │ │ │ │   │ │ ├── https://deno.land/std@0.66.0/fs/_util.ts *
│ │ │ │ │   │ │ └── https://deno.land/std@0.66.0/fs/ensure_dir.ts *
│ │ │ │ │   │ ├─┬ https://deno.land/std@0.66.0/fs/ensure_link.ts (1.57KB)
│ │ │ │ │   │ │ ├── https://deno.land/std@0.66.0/path/mod.ts *
│ │ │ │ │   │ │ ├── https://deno.land/std@0.66.0/fs/_util.ts *
│ │ │ │ │   │ │ ├── https://deno.land/std@0.66.0/fs/ensure_dir.ts *
│ │ │ │ │   │ │ └── https://deno.land/std@0.66.0/fs/exists.ts (708B)
│ │ │ │ │   │ ├─┬ https://deno.land/std@0.66.0/fs/ensure_symlink.ts (2.01KB)
│ │ │ │ │   │ │ ├── https://deno.land/std@0.66.0/path/mod.ts *
│ │ │ │ │   │ │ ├── https://deno.land/std@0.66.0/fs/_util.ts *
│ │ │ │ │   │ │ ├── https://deno.land/std@0.66.0/fs/ensure_dir.ts *
│ │ │ │ │   │ │ └── https://deno.land/std@0.66.0/fs/exists.ts *
│ │ │ │ │   │ ├── https://deno.land/std@0.66.0/fs/eol.ts (714B)
│ │ │ │ │   │ ├── https://deno.land/std@0.66.0/fs/exists.ts *
│ │ │ │ │   │ ├─┬ https://deno.land/std@0.66.0/fs/expand_glob.ts (7.44KB)
│ │ │ │ │   │ │ ├── https://deno.land/std@0.66.0/_util/assert.ts *
│ │ │ │ │   │ │ ├── https://deno.land/std@0.66.0/path/mod.ts *
│ │ │ │ │   │ │ └─┬ https://deno.land/std@0.66.0/fs/walk.ts (4.35KB)
│ │ │ │ │   │ │   ├── https://deno.land/std@0.66.0/_util/assert.ts *
│ │ │ │ │   │ │   └── https://deno.land/std@0.66.0/path/mod.ts *
│ │ │ │ │   │ ├─┬ https://deno.land/std@0.66.0/fs/move.ts (1.43KB)
│ │ │ │ │   │ │ ├── https://deno.land/std@0.66.0/fs/_util.ts *
│ │ │ │ │   │ │ └── https://deno.land/std@0.66.0/fs/exists.ts *
│ │ │ │ │   │ ├── https://deno.land/std@0.66.0/fs/read_json.ts (798B)
│ │ │ │ │   │ ├── https://deno.land/std@0.66.0/fs/walk.ts *
│ │ │ │ │   │ └── https://deno.land/std@0.66.0/fs/write_json.ts (1.54KB)
│ │ │ │ │   └── https://deno.land/std@0.66.0/path/mod.ts *
│ │ │ │ ├── https://deno.land/x/eta@v1.12.2/polyfills.ts (294B)
│ │ │ │ └─┬ https://deno.land/x/eta@v1.12.2/utils.ts (2.78KB)
│ │ │ │   ├── https://deno.land/x/eta@v1.12.2/config.ts *
│ │ │ │   └── https://deno.land/x/eta@v1.12.2/polyfills.ts *
│ │ │ └── https://deno.land/x/eta@v1.12.2/polyfills.ts *
│ │ ├─┬ https://deno.land/x/eta@v1.12.2/containers.ts (278B)
│ │ │ ├── https://deno.land/x/eta@v1.12.2/compile.ts *
│ │ │ └─┬ https://deno.land/x/eta@v1.12.2/storage.ts (758B)
│ │ │   └── https://deno.land/x/eta@v1.12.2/utils.ts *
│ │ ├── https://deno.land/x/eta@v1.12.2/err.ts *
│ │ ├── https://deno.land/x/eta@v1.12.2/storage.ts *
│ │ └── https://deno.land/x/eta@v1.12.2/utils.ts *
│ ├─┬ https://deno.land/x/eta@v1.12.2/parse.ts (5.78KB)
│ │ ├── https://deno.land/x/eta@v1.12.2/config.ts *
│ │ ├── https://deno.land/x/eta@v1.12.2/err.ts *
│ │ └── https://deno.land/x/eta@v1.12.2/utils.ts *
│ └── https://deno.land/x/eta@v1.12.2/parse.ts *
├── https://deno.land/x/eta@v1.12.2/compile.ts *
├── https://deno.land/x/eta@v1.12.2/config.ts *
├── https://deno.land/x/eta@v1.12.2/containers.ts *
├── https://deno.land/x/eta@v1.12.2/file-handlers.ts *
├─┬ https://deno.land/x/eta@v1.12.2/file-helpers.ts (504B)
│ ├── https://deno.land/x/eta@v1.12.2/config.ts *
│ └── https://deno.land/x/eta@v1.12.2/file-handlers.ts *
├── https://deno.land/x/eta@v1.12.2/parse.ts *
└─┬ https://deno.land/x/eta@v1.12.2/render.ts (3.87KB)
  ├── https://deno.land/x/eta@v1.12.2/compile.ts *
  ├── https://deno.land/x/eta@v1.12.2/compile.ts *
  ├── https://deno.land/x/eta@v1.12.2/config.ts *
  ├── https://deno.land/x/eta@v1.12.2/config.ts *
  ├── https://deno.land/x/eta@v1.12.2/err.ts *
  ├── https://deno.land/x/eta@v1.12.2/file-handlers.ts *
  └── https://deno.land/x/eta@v1.12.2/polyfills.ts *
```

**Comment by shadowtime2000** [comment]
Hmm it seems the build process may have glitched or something so the changes weren't included?? However once I do the release of v1rt1ls changes to update for latest Deno version it should be included in the release then....

**Comment by shadowtime2000** [comment]
Okay as an update it seems the problem lied in my own handling of Git, where there was some conflicts in merges I never fixed or something.

**Comment by shadowtime2000** [comment]
Closed by 1.12.3
